lib/OnboardingMessageProvider.jsm

```diff
@@ -73,7 +72,7 @@ const ONBOARDING_MESSAGES = () => ([
         data: {url: "https://addons.mozilla.org/en-US/firefox/addon/ghostery/"},
       },
     },
-    targeting: "isInExperimentCohort == 2",
+    targeting: "providerCohorts.onboarding == 'ghostery'",
```

Comment:
We should probably let someone know about this if they want to experiment on this more? maybe just tell shell or something in the next meeting